examples/local_pow.py

```diff
@@ -1,6 +1,4 @@
-from __future__ import absolute_import, division, print_function, \
-    unicode_literals
-
+# coding=utf-8
```

Comment:
❔ Is this still needed?  We might be able to remove these; I think Python 3 assumes UTF-8 encoding..

Reply:
Indeed, will remove it.